generate_quick_coverage.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Generate coverage by running test directories individually and merging them
+mkdir -p coverage_reports
+dirs=$(find test -maxdepth 1 -mindepth 1 -type d)
```

Shellcheck (0.11.0)

[warning] dirs appears unused. Verify use (or export if used externally).

(SC2034)